src/pages/Documents.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useContext, useEffect, useState } from 'react';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { useLocation } from 'react-router-dom';
+import { useTheme } from '@emotion/react';
+import { useMediaQuery } from '@mui/material';
+import { ConfirmationModal, SetAclPermissionsModal, UploadDocumentModal } from '@components/Modals';
+import { useNotification, useSession } from '@hooks';
+import { DocumentListContext } from '@contexts';
+import { truncateText } from '@utils';
+import { DocumentTable } from '@components/Documents';
+import { Container } from '@mui/system';
```

Comment:
Is there a reason Container is imported from '@mui/system' rather than '@mui/material' like it is elsewhere?

Reply:
No reason, it looks like the import path auto completed to the wrong path

I'll change it back